HaulageApplication/Haulage/BaseClasses/BillingHandler/Billing.cs

```diff
@@ -8,20 +9,24 @@ namespace Haulage.BaseClasses.BillingHandler
 {
     public class Billing
     {
-        public Guid BillID;
-        public Guid customerID;
-        public float totalAmount;
-        enum Billingstatus
-        {
-            notPaid,
-            paid
-
-        }
+       public int BillId { get; set; }
+       public string Fullname { get; set; }
+       public string Email { get; set; }
+       public string Item {  get; set; }
+       public string ItemDesc { get; set; }
+       public string Cost { get; set; }
```

Comment:
why is a cost of type string? would it make better use of the type system to have it as something like a decimal, which is "commonly used in financial applications, for currency amounts" 
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/floating-point-numeric-types

Reply:
changed cost from string to float and changed from text to decimal in sqlite table